instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -449,17 +449,27 @@ class OpenTelemetryMiddleware:
 
     Args:
         app: The ASGI application callable to forward requests to.
+        excluded_urls: (opentelemetry.util.http.ExcludeList) Optional parameter to specify URLs that should be excluded from tracing. 
```

Comment:
It would be probably metter to add the type annotation instead of writing the type here

Reply:
I considered that, but I'm unsure of the implications of that - it seemed like adding type annotations could impact existing implementations. I'm pretty new to this library and I didn't want to push actual code changes until I'm more familiar with it.

Reply:
Updated to use type hinting instead of comments